gdi/opentelemetry/deployments/deployments-windows-puppet.rst

```diff
@@ -45,7 +39,7 @@ The class accepts the parameters described in the following table:
      - Description
      - Default value
    * - ``collector_version``
-     - Version of the Collector package to install, for example, ``0.25.0``. The version should correspond to :new-page:`Github Releases <https://github.com/signalfx/splunk-otel-collector/releases>` without the preceding ``v``. This attribute is required.
+     - Version of the Collector package to install, for example, ``0.25.0``. The version musts correspond to :new-page:`Github Releases <https://github.com/signalfx/splunk-otel-collector/releases>` without the preceding ``v``. This attribute is required.
```

Comment:
```suggestion
     - Version of the Collector package to install, for example, ``0.25.0``. The version must correspond to :new-page:`Github Releases <https://github.com/signalfx/splunk-otel-collector/releases>` without the preceding ``v``. This attribute is required.
```

Reply:
Could also be `The version must correspond to a Github Release`, which may be more accurate.